config/initializers/barong.rb

```diff
@@ -112,6 +116,14 @@
   config.set(:kycaid_authorization_token, '')
   config.set(:kycaid_sandbox_mode, 'true', type: :bool)
   config.set(:kycaid_api_endpoint, 'https://api.kycaid.com/')
+
+  # Auth0 configuration -----------------------------------------------
+  config.set(:auth0_tenant_address, 'openware-barong.us.auth0.com')
+  config.set(:client_id, ENV['CLIENT_ID'])
```

Comment:
We should prefix those variables.
If this is specific to Auth0 let's prefix with AUTH0
Is this code compatible with OAuth protocol? In this case we would prefix with OAUTH.